### PR TITLE
(MODULES-8615) Fix rules with ipvs not parsing

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

**Comment by eimlav** [comment]
Adhoc ran sucessfully:
Screen Shot 2019-03-25 at 11 51 33

File: spec/acceptance/firewall_attributes_happy_path_spec.rb [hunk 1]
```diff
@@ -350,6 +349,11 @@ class { '::firewall': }
             chain          => 'OUTPUT',
             table          => 'mangle',
           }
+          firewall { '1002 - set ipvs':
```
Comment (mateusz-gozdek-sociomantic) [inline spec/acceptance/firewall_attributes_happy_path_spec.rb]: How this test tests ipvs?

Comment (eimlav) [reply spec/acceptance/firewall_attributes_happy_path_spec.rb]: @mateusz-gozdek-sociomantic Nicely spotted! Tt seems I forgot to actually add the full test. I've amended this in my PR #833 and fixed an idempotency issue I uncovered.

Comment (mateusz-gozdek-sociomantic) [reply spec/acceptance/firewall_attributes_happy_path_spec.rb]: You're welcome, cool that it's fixed now!

**Pull request merged**